docs/software_requirements/timers.sdoc

```diff
@@ -26,12 +23,334 @@ UID: ZEP-SRS-4-2
 STATUS: Draft
 TYPE: Functional
 COMPONENT: Timers
-TITLE: Call functions in interrupt context
+TITLE: Timer definition at compile time
+STATEMENT: >>>
+The Zephyr RTOS shall provide a mechanism to define and initialize a timer at compile time.
+<<<
+RELATIONS:
+- TYPE: Parent
+  VALUE: ZEP-SYRS-18
+
+[REQUIREMENT]
+UID: ZEP-SRS-4-3
+STATUS: Draft
+TYPE: Non-Functional
+COMPONENT: Timers
+TITLE: Timer status
+STATEMENT: >>>
+A Zephyr RTOS timer shall have a status which indicates the expiration of the timer.
```

Comment:
If it would speak about a status field here I would agree that it should be in the design, but in this case we only speak of a status a timer has if it is expired or not. Obviously that will most of the time end up in a design with a status filed 😄